cmd/outputs.go

```diff
@@ -30,7 +30,13 @@ func getAllOutputConstructors() (map[string]output.Constructor, error) {
 			return nil, errors.New("the kafka output was deprecated in k6 v0.32.0 and removed in k6 v0.34.0, " +
 				"please use the new xk6 kafka output extension instead - https://github.com/k6io/xk6-output-kafka")
 		},
-		"statsd": statsd.New,
+		"statsd": func(params output.Params) (output.Output, error) {
+			params.Logger.Warn("The statsd output is deprecated, and will be removed in a future k6 version. " +
```

Comment:
We probably should update the `datadog` case too since it recommends using this `statsd` :sweat_smile:

Reply:
I thought about this but prefer to leave it for when we remove this - to update both to the extension. Or to be honest to just drop kafka and datadog as they have not been a thing for over 2 years at this point :sweat_smile:

Reply:
:+1: for dropping (but OK with doing this in a separate PR)